<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wAddPath</class>
 <widget class="QMainWindow" name="wAddPath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wAddPath</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="lblID">
      <property name="text">
       <string notr="true">ID:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="ledID">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string notr="true">none</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblPath">
      <property name="text">
       <string notr="true">Path:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="ledPath"/>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="cbxRedirect">
      <property name="text">
       <string notr="true">Path redirect:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cobxRedirect"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lblTerms">
      <property name="text">
       <string notr="true">Terms:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string notr="true">Save</string>
      </property>
      <property name="icon">
       <iconset resource="../../ressources.qrc">
        <normaloff>:/rec/data/icons/iconCheck.svg</normaloff>:/rec/data/icons/iconCheck.svg</iconset>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QTableView" name="tvwTerms">
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
